resources/shaders/glbillbshader.frag

```diff
@@ -14,12 +15,24 @@ struct FogParam {
 };
 
 uniform sampler2D texture0;
+uniform usamplerBuffer palbuf;
+uniform bool repaint;
 uniform FogParam fog;
 
 float getFogRatio(FogParam fogpar, float dist);
 
 void main() {
-    vec4 fragcol = texture(texture0, texuv) * colour;
+    vec4 fragcol = texture(texture0, texuv);
+    uint index = uint(fragcol.r * 255);
+    uvec4 newcol = texelFetch(palbuf, int( 256 * paletteid + index));
+
+   if (repaint == true) {
+	if (index > 0)
+	    fragcol = vec4(newcol.r / 255.0, newcol.g / 255.0, newcol.b / 255.0, 1.0);
```

Comment:
560ti under linux doesn't like division of uvec4 and saving it into vec4:
```
0(31) : error C1020: invalid operands to "/"
0(31) : error C1020: invalid operands to "/"
0(31) : error C1020: invalid operands to "/"
(0) : error C2003: incompatible options for link
```
changing newcol to vec4 fix this.

Reply:
will go vec4 for ease